extension/package.json

```diff
@@ -1473,7 +1473,7 @@
       },
       {
         "view": "dvc.views.welcome",
-        "contents": "The extension is currently unable to initialize.\n[Show Setup](command:dvc.showDvcSetup)",
+        "contents": "The extension is currently unable to initialize.\n[Show Setup](command:dvc.showDvcSetup)\nNew to DVC? Check out [dvc.org](https://dvc.org/) to learn more or quickly try the extension with our [demo](https://github.com/iterative/vscode-dvc-demo).",
```

Comment:
[C] On top of changing the project I think the order here should be reversed. Link to project first and docs second. It would be good to get a link to a video demonstrating the extension in here too. @shcheklein WDYT about adding a link to Alex's "Best MLOps Practices for Building End-to-End Machine Learning Computer Vision Projects" video at the timestamp where he starts showing the extension? E.g https://youtu.be/E26IaD7bNXg?t=1135

Reply:
[Q] Is this obvious enough?

Reply:
Link to the video sounds like a good idea. A lot of people will find it useful and easier to get the idea vs looking into the demo project.

Reply:
Good ideas! Updated the text with a video link and put docs link at the end instead of in the beginning.